lib/comment/comment.models.js

```diff
@@ -0,0 +1,27 @@
+const Joi = require('joi');
+
+module.exports = {
+  db: (sequelize, Sequelize) => {
+    const Comment = sequelize.define('Comment', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true
+      },
+      body: Sequelize.TEXT,
+    });
+
+    Comment.associate = function (models) {
+      models.Comment.belongsTo(models.Post, {as: 'Post'});
```

Comment:
The 'as: Post' part is a relationship alias, but since it's the same as the model name you are relating, it shouldn't be needed.